Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -82,6 +82,18 @@ ErrorOr<Optional<ReadonlyBytes>> PNGImageDecoderPlugin::icc_data()
     return OptionalNone {};
 }
 
+static void log_png_error(png_structp png, char const* error_message)
+{
+    (void)png;
```

Comment:
You can void needing to cast the argument to void here if you don't include the variable name after the argument.